hack/crds-to-consul-helm/main.go

```diff
@@ -0,0 +1,87 @@
+// Script to move generated CRD yaml into consul-helm and modify it to match
+// the expected consul-helm format.
```

Comment:
just a thought but it might be work mentioning somewhere in here that the binary (appears to be) expects to be run from CWD and not from top level consul-helm? Maybe I'm not reading the code correctly

Reply:
Yeah thanks for calling this out. I think the better pattern is to call this through the `Makefile`. I've pushed a new commit with that change. Then you can run `make ctrl-crd-copy helm=../consul-helm` and it will work.

Reply:
Ah fantastic idea, love it!